File: 07.10-Project/BackendApp/Models/OrderModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace BackendApp.Models
{
    public class Order
    {
        public int Id { get; set; }
        public int? CustomerId { get; set; }
        public int? EmployeeId { get; set; }
        public DateTime? OrderDate { get; set; }
        public int MedicamentId { get; set; }
        public int Qty { get; set; }
        [Column(TypeName = "decimal")]
        public decimal Price { get; set; }
        public  Customer Customer { get; set; }
        public  Employee Employee { get; set; }

        public Order()
        {
            MedicamentOrders = new List<MedicamentOrderModel>();
        }
        public List<MedicamentOrderModel> MedicamentOrders { get; set; }
    }
}

File: 07.10-Project/FrontendApp/src/app/models/order.model.ts
import { Customer } from "./customer.model";
import { Employee } from "./employee.model";
import { MedicamentOrder } from "./medicament-order.model";

export interface Order {
    id: number;
    customerId: number;
    employeeId: number;
    orderDate: Date;
    medicamentId: number;
    qty: number;
    price: number;
    customers: Customer[];
    employee: Employee[];

   // medicamentOrders: MedicamentOrder[];
    //public Order()  { medicamentOrders }

  //     {  MedicamentOrders = new List<MedicamentOrderModel>();  }
  // public List < MedicamentOrderModel > MedicamentOrders { get; set; }

}
